src/btree/bt_read.c

```diff
@@ -579,7 +579,7 @@ __page_read(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
 		    session, ref->page_las->las_pageid));
 
 	WT_REF_SET_STATE(ref, final_state);
-	return (ret);
```

Comment:
I don't see how we could come here with ret anything other than `0`, and I think it would be bad if we did. Could you add an assert that `ret == 0` instead or as well?

Reply:
> Could you add an assert that ret == 0 instead or as well?

Sure, pushed in 9d65fc85a.